src/helpers/authentication.js

```diff
@@ -85,30 +85,37 @@ module.exports = (application) => {
   };
 
   const refreshToken = (user) => {
-    if (user === 'default') return;
```

Comment:
What does `user === 'default'` mean anyway?

Reply:
`user === 'default'` means authenticating as the oauth application owner via client credentials.. it is used when decoding encoded token send with websocket..